core/proto/objects.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+option go_package = "github.com/mekstack/nataas/core/proto";
+
+
+package objects;
+
+// Route model
+message Route {
+  string target_ip = 1;
+  uint32 port = 2;
+  repeated Subdomain subdomains = 3;
+}
```

Comment:
why is subdomain repeated? 
the dependency is the other way around: subdomains have target_ip, and only a single one